[2024-08-12T09:39:20.196+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-12T09:39:20.216+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-12T09:39:03.023364+00:00 [queued]>
[2024-08-12T09:39:20.221+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-12T09:39:03.023364+00:00 [queued]>
[2024-08-12T09:39:20.221+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-08-12T09:39:20.231+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): limpieza> on 2024-08-12 09:39:03.023364+00:00
[2024-08-12T09:39:20.239+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_prueba_tecnica', 'limpieza', 'manual__2024-08-12T09:39:03.023364+00:00', '--job-id', '236', '--raw', '--subdir', 'DAGS_FOLDER/dag_prueba_tecnica.py', '--cfg-path', '/tmp/tmpvuk5rxav']
[2024-08-12T09:39:20.239+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=174) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-12T09:39:20.240+0000] {standard_task_runner.py:91} INFO - Job 236: Subtask limpieza
[2024-08-12T09:39:20.240+0000] {standard_task_runner.py:64} INFO - Started process 187 to run task
[2024-08-12T09:39:20.283+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-12T09:39:03.023364+00:00 [running]> on host 0ad27a48216c
[2024-08-12T09:39:20.364+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='miguel_guimarey' AIRFLOW_CTX_DAG_ID='dag_prueba_tecnica' AIRFLOW_CTX_TASK_ID='limpieza' AIRFLOW_CTX_EXECUTION_DATE='2024-08-12T09:39:03.023364+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-12T09:39:03.023364+00:00'
[2024-08-12T09:39:20.365+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-12T09:39:23.077+0000] {warnings.py:112} WARNING - /opt/***/dags/dag_prueba_tecnica.py:42: FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  df = pd.read_json(data, orient='records')

[2024-08-12T09:39:33.439+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-08-12T09:40:08.909+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-12T09:40:08.910+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-12T09:40:08.918+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_prueba_tecnica, task_id=limpieza, run_id=manual__2024-08-12T09:39:03.023364+00:00, execution_date=20240812T093903, start_date=20240812T093920, end_date=20240812T094008
[2024-08-12T09:40:09.072+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-12T09:40:09.086+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-12T09:40:09.090+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
